ilastik/applets/serverConfiguration/configStorage.py

```diff
@@ -80,58 +77,13 @@ def get_server(self, id_) -> typing.Optional[types.ServerConfig]:
 
         return None
 
-    def _server_as_dict(self, srv):
-        res = attr.asdict(srv)
-        res["autostart"] = str(int(res.get("autostart", False)))
-        return res
-
-    def _serialize_device(self, device):
-        device_str = f"{device['id']}::{device['name']}"
-        if device["enabled"]:
-            device_str += "::enabled"
-        return device_str
-
-    def _serialize_devices(self, devices):
-        result = []
-
-        for dev in devices:
-            try:
-                result.append(self._serialize_device(dev))
-            except Exception:
-                pass
-
-        if len(result) > 1:
-            result.insert(0, '')
-
-        return '\n'.join(result)
-
-    def _write_server_entry(self, srv):
-        section_id = f"{self.PREFIX}{srv.id}"
-
-        self._config.add_section(section_id)
-        for key, value in self._server_as_dict(srv).items():
-            if key == "id":
-                continue
-
-            if key == "devices":
-                value = self._serialize_devices(value)
-
-            self._config.set(section_id, key, value)
-
     def store(self, servers) -> None:
-        current_servers = self.get_servers()
-
-        for srv in current_servers:
-            self._config.remove_section(f"{self.PREFIX}{srv.id}")
+        servers_data = []
 
         for srv in servers:
-            self._write_server_entry(srv)
+            servers_data.append(attr.asdict(srv))
 
-        if isinstance(self._dst, str):
-            with open(self._dst, 'w+') as out:
-                self._config.write(out)
-        else:
-            self._config.write(self._dst)
+        preferences.set(SERVER_CONFIG_GROUP, SERVER_CONFIG_LIST_KEY, servers_data)
```

Comment:
the cool thing is, that this fixes #2355 :+1:  (which the previously installed write to .ilastikrc it originally caused :/)